src/plots/polar/legacy/area_attributes.js

```diff
@@ -10,15 +10,48 @@
 
 var scatterAttrs = require('../../../traces/scatter/attributes');
 var scatterMarkerAttrs = scatterAttrs.marker;
+var extendFlat = require('../../../lib/extend').extendFlat;
+
+var deprecationWarning = 'Area traces are deprecated!';
 
 module.exports = {
-    r: scatterAttrs.r,
-    t: scatterAttrs.t,
+    r: extendFlat({}, scatterAttrs.r, {
```

Comment:
Follow-up on https://github.com/plotly/plotly.js/pull/3023/commits/1c0d3ed0512ef826bf2e6dc9d576bccfff2f18f3

Reply:
Ah great, nice pattern for adding these warnings!

`scatterAttrs.r` already [asks users to switch to `scatterpolar`](https://github.com/plotly/plotly.js/blob/2a3dc2656d3c832ceb24b3c22c9155aa02fd7c7a/src/traces/scatter/attributes.js#L548) - so here we should switch that so the deprecation warning points people to `barpolar`. Similarly for the layout attrs, can we point people to `polar` subplots?

Reply:
new attempt: https://github.com/plotly/plotly.js/pull/3023/commits/019bacfc88069f51133f624c49063e5b01c94d58